src/modules/checkitem.js

```diff
@@ -22,7 +22,7 @@ const checkItem = () => {
     const iconImg = document.querySelectorAll('.icon');
     const clearBtn = document.querySelector('#clear-btn');
     const storageData = JSON.parse(localStorage.getItem('taskList'));
-    // Strike the item, show trash icon,and change the value of completed when checked
+    // Strike the item, show trash icon, and change the value of completed when checked
     for (let i = 0; i < checkbox.length; i += 1) {
       if (checkbox[i].checked) {
         listInfo[i].classList.add('line-through');
```

Comment:
Kindly use long and descriptive names. Considering that it represents a certain behavior, the variable name should say what they do so that anyone can easily understand your code. for example, instead of saying `listInfo` you can say `listText` and also instead of using `editCont` you can say `editContainer`